src/lib/list/list.scss

```diff
@@ -23,12 +24,13 @@ $mat-dense-base-height: 40px;
 $mat-dense-avatar-height: 48px;
 $mat-dense-two-line-height: 60px;
 $mat-dense-three-line-height: 76px;
+$mat-dense-multi-line-padding: 16px;
```

Comment:
Note that the dense padding here is the same as the regular one, however the spec doesn't seem to mention exactly how to handle more than three lines. I based this number on the spacing between the text and the element top for three-line items.